devdoc/constbuffer_array_splitter_requirements.md

```diff
@@ -13,6 +13,10 @@ For example, it is used to upload large payloads to Blob store, split across lar
 MOCKABLE_FUNCTION(, CONSTBUFFER_ARRAY_HANDLE, constbuffer_array_splitter_split, CONSTBUFFER_ARRAY_HANDLE, buffers, uint32_t, max_buffer_size);
 ```
 
+```c
```

Comment:
nit: remove, can keep these all in the same code block

Reply:
removed